internal/cli/atlas/deployments/setup.go

```diff
@@ -493,6 +499,14 @@ func (opts *SetupOpts) runConnectWith(cs string) error {
 	switch opts.connectWith {
 	case skipConnect:
 		fmt.Fprintln(os.Stderr, "connection skipped")
+	case compassConnect:
+		if !compass.Detect() {
+			return errors.New("MongoDB Compass not found in your system")
```

Comment:
can you make a sentinel error?

Reply:
for what purpose, being reused in the future? isn't this early optimization?